app/models/samples/json/model_with_humidty_setpoints.json

```diff
@@ -0,0 +1,1588 @@
+                                "type": "ScheduleDay",
+                                "name": "Hospital BLDG_LIGHT_EXTD_SCH_Wkdy",
+                                "values": [
+                                    0.5,
+                                    0.9,
+                                    0.5
+                                ],
+                                "times": [
+                                    [
+                                        0,
+                                        0
+                                    ],
+                                    [
+                                        8,
+                                        0
+                                    ],
+                                    [
+                                        16,
+                                        0
+                                    ]
+                                ],
+                                "interpolate": false
+                            },
+                            "apply_sunday": false,
+                            "apply_monday": true,
+                            "apply_tuesday": true,
+                            "apply_wednesday": true,
+                            "apply_thursday": true,
+                            "apply_friday": true,
+                            "apply_saturday": false,
+                            "apply_holiday": false,
+                            "start_date": [
+                                1,
+                                1
+                            ],
+                            "end_date": [
+                                12,
+                                31
+                            ]
+                        }
+                    ],
+                    "summer_designday_schedule": {
+                        "type": "ScheduleDay",
+                        "name": "Hospital BLDG_LIGHT_EXTD_SCH_SmrDsn",
+                        "values": [
+                            1.0
+                        ],
+                        "times": [
+                            [
+                                0,
+                                0
+                            ]
+                        ],
+                        "interpolate": false
+                    },
+                    "winter_designday_schedule": {
+                        "type": "ScheduleDay",
+                        "name": "Hospital BLDG_LIGHT_EXTD_SCH_WntrDsn",
+                        "values": [
+                            0.0
+                        ],
+                        "times": [
+                            [
+                                0,
+                                0
+                            ]
+                        ],
+                        "interpolate": false
+                    },
+                    "schedule_type_limit": "Fractional"
+                },
+                {
+                    "type": "ScheduleRulesetAbridged",
```

Comment:
@chriswmackey I got a ValidationError in this section.

Reply:
What is the error?

Reply:
`schedules -> 9 -> type`
`value is not a valid enumeration member (type=type_error.enum)`
`schedules -> 9 -> start_date`
`field required (type=value_error.missing)`
`schedules -> 9 -> values`
`field required (type=value_error.missing)`

Reply:
Are you running it with an old version of the schema? There don't seem to be anything wrong with the JSON.